packages/itwin/breakdown-trees/src/BreakdownTrees.ts

```diff
@@ -57,7 +62,12 @@ export class BreakdownTrees {
    * @internal
    */
   public static translate(key: string | string[], options?: LocalizationOptions): string {
-    return BreakdownTrees.i18n.getLocalizedStringWithNamespace(BreakdownTrees.i18nNamespace, key, options);
+    if (Array.isArray(key)) {
```

Comment:
is this right? i dont think your string concat does what you want it to. re-read what you wrote

Reply:
fixed